Sources/TuistEnvKit/Commands/CommandRunner.swift

```diff
@@ -101,6 +101,8 @@ class CommandRunner: CommandRunning {
     }
 
     func runVersion(_ version: String) throws {
+        _ = Version(stringLiteral: version)
```

Comment:
I would rather make it explicit
```suggestion
        guard Version(string: version) != nil else {
            logger.error("\(version) is not a valid version")
            exiter(1)
        }
```

Reply:
That's a good suggestion. Thanks😉

Reply:
> I would rather make it explicit

Can you re-review my code?
I changed the code like yours.
And I think it's better.